packages/beacon-node/test/spec/presets/index.test.ts

```diff
@@ -62,7 +62,16 @@ specTestIterator(
     rewards: {type: RunnerType.default, fn: rewards},
     sanity: {type: RunnerType.default, fn: sanity},
     shuffling: {type: RunnerType.default, fn: shuffling},
-    ssz_static: {type: RunnerType.custom, fn: sszStatic()},
+    ssz_static: {
```

Comment:
these types are disabled momentarily as there is LightClientHeader introduced in this spec release and is being addressed in separate PR which will fix and reenable all these types